modules/cockroachdb/cockroachdb.go

```diff
@@ -233,6 +239,46 @@ func addTLS(ctx context.Context, container testcontainers.Container, opts option
 	return nil
 }
 
+// setRecommendedSettings applies the cluster settings recommended by cockroachlabs for testing clusters.
+// See https://www.cockroachlabs.com/docs/stable/local-testing for more information.
+func setRecommendedSettings(ctx context.Context, container testcontainers.Container, opts options) error {
+	port, err := container.MappedPort(ctx, defaultSQLPort)
+	if err != nil {
+		return err
+	}
+
+	host, err := container.Host(ctx)
+	if err != nil {
+		return err
+	}
+
+	db, err := sql.Open("pgx/v5", connString(opts, host, port))
+	if err != nil {
+		return err
+	}
+	defer db.Close()
+
+	stmts := []string{
+		"SET CLUSTER SETTING kv.range_merge.queue_interval = '50ms'",
+		"SET CLUSTER SETTING jobs.registry.interval.gc = '30s'",
+		"SET CLUSTER SETTING jobs.registry.interval.cancel = '180s'",
+		"SET CLUSTER SETTING jobs.retention_time = '15s'",
+		"SET CLUSTER SETTING sql.stats.automatic_collection.enabled = false",
+		"SET CLUSTER SETTING kv.range_split.by_load_merge_delay = '5s'",
+		`ALTER RANGE default CONFIGURE ZONE USING "gc.ttlseconds" = 600`,
+		`ALTER DATABASE system CONFIGURE ZONE USING "gc.ttlseconds" = 600`,
+	}
+
+	for _, stmt := range stmts {
+		_, err = db.Exec(stmt)
+		if err != nil {
+			return err
```

Comment:
suggestions: let's wrap the error to provide more context on potential errors
```suggestion
			return fmt.Errorf("db.Exec: %w", err)
```

Reply:
fixed